types/lhr/audit-details.d.ts

```diff
@@ -58,7 +58,9 @@ declare module Details {
 
   interface List {
     type: 'list';
-    items: SnippetValue[]
+    // NOTE: any `Details` type *should* be usable in `items`, but check
+    // styles/report-ui-features are good before adding.
```

Comment:
also added a note that in theory any details should work in a list, it's just repeatedly adding elements in the `lh-details` div, but in practice they may interfere with each other due to styling, or assumptions in details renderer (e.g. they're the only child in their parent), etc. Need to double check they work before adding another.